public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_monitor/azure_monitor_datasource.ts

```diff
@@ -251,7 +255,11 @@ export default class AzureMonitorDatasource extends DataSourceWithBackend<AzureM
     );
     return this.getResource(url)
       .then((result: AzureMonitorMetricNamespacesResponse) => {
-        return ResponseParser.parseResponseValues(result, 'name', 'properties.metricNamespaceName');
+        return ResponseParser.parseResponseValues(
+          result,
+          'properties.metricNamespaceName',
```

Comment:
small change here but it affects other parts of the plugin. Before we we using as label the metric namespace "name" which was something like `microsoft.insights-components` but I think it's more familiar to use the metric namespace name: `microsoft.insights/components`. (This is just a cosmetic change)

Reply:
Agree with this change! 💯